src/commands/generate.rs

```diff
@@ -68,10 +61,3 @@ fn command(name: &str, binary_path: PathBuf, args: &[&str], project_type: &Targe
     c.args(args);
     c
 }
-
-fn get_target_type(template: &str) -> TargetType {
-    if template.contains("rust") {
-        return TargetType::Rust;
-    }
-    TargetType::default()
-}
```

Comment:
this is the highlight of my career

Reply:
throwback ![image](https://user-images.githubusercontent.com/9408157/66166591-5b22ba80-e5fd-11e9-9cd0-d5feea71474b.png)